trlx/utils/__init__.py

```diff
@@ -86,39 +88,42 @@ class OptimizerName(str, Enum):
     SGD: str = "sgd"
 
 
-def get_optimizer_class(name: OptimizerName):
+torch_optimizers: Dict[str, type] = dict(
+    inspect.getmembers(
+        torch.optim, lambda x: inspect.isclass(x) and x.__name__ != "Optimizer"
+    )
+)
+torch_optimizers = {
+    name.lower(): optimizer for name, optimizer in torch_optimizers.items()
+}
```

Comment:
This is a tad hacky but cool. If we do this then there's little point in having the OptimizerName enum other than explicitly laying out what optimizers are available.